packages/types/lib/driver.ts

```diff
@@ -44,6 +44,19 @@ export interface SessionCommands {
   getSession(): Promise<SingularSessionData>;
 }
 
+export interface ExecuteCommands {
+  executeOverload(script: string, args: any[]): Promise<any>;
```

Comment:
This signature should match the one [here](https://github.com/appium/appium/pull/17321/files#diff-68bc5501503dcc37a3bc53d4dbd921a1f7d02f7e015f330aa0e9b14b3ed1e880R19), yes?

Reply:
yeah, fixing